README.md

```diff
@@ -1,7 +1,67 @@
 # Kamu CLI
 
+[![Build Status](https://travis-ci.org/kamu-data/kamu-cli.svg?branch=master)](https://travis-ci.org/kamu-data/kamu-cli)
+
 [Installation](docs/install.md)
 
 [How it Works](docs/architecture.md)
 
 [Developer Guide](docs/developer_guide.md)
+
+Step by step guide:
```

Comment:
Can you please merge this info with existing developer guide, there's a lot of duplication with what's already in there. Also I don't think this stuff belongs in the readme as it's only relevant for developers/contributors.